tools/pythonpkg/scripts/conftest.py

```diff
@@ -0,0 +1,108 @@
+import pathlib
+import pytest
+import typing
+import warnings
+import skipped_tests
+
+SQLLOGIC_TEST_CASE_NAME = "test_sqllogic"
+SQLLOGIC_TEST_PARAMETER = "test_script_path"
+
+def pytest_addoption(parser: pytest.Parser):
+    parser.addoption(
+        "--tests-dir", required=True, type=pathlib.Path, dest="duckdb_tests_dir", help="Path to the DuckDB tests directory"
+    )
+    parser.addoption(
+        "--start-offset", type=int, dest="start_offset", help="Index of the first test to run"
+    )
+    parser.addoption(
+        "--end-offset", type=int, dest="end_offset", help="Index of the last test to run"
+    )
+
+def pytest_keyboard_interrupt(excinfo: pytest.ExceptionInfo):
+    #TODO: CTRL+C interrupts DuckDB queries and does not go immediately to pytest
+    pytestmark = pytest.mark.skip(reason="Keyboard interrupt")
+
+def pytest_configure(config: pytest.Config):
+    # TODO: Change working directory to temp file. Store original working directory.
```

Comment:
I guess you want to do this to enable relative paths in test files?
I don't think the original python sqllogictester does this, so I wouldn't make this a blocker for this PR